packages/generator/README.md

```diff
@@ -146,6 +146,15 @@ const config = {
 
 When the specified amount of memory is reached, Redis can be configured to apply an eviction policy. Refer to [this article](https://redis.io/topics/lru-cache/) to setup Redis as an LRU cache and to learn more about supported eviction policies. 
 
+## Logging
+Default log level of the SDK is WARN (warning). SDK uses [loglevel](https://www.npmjs.com/package/loglevel) npm package. All the log levels supported by [loglevel](https://www.npmjs.com/package/loglevel) package are supported in SDK.
+
+To change the loglevel, import sdkLogger and set the level on it.
+```javascript
+import { sdkLogger } from "@commerce-apps/core"
```

Comment:
[ISSUE] We don't have customers using core for anything else, we shouldn't here either. Suggest changing to

```javascript
loglevel.getLogger("COMMERCER-SDK").setLevel(INFO);
```

Alternatively, you can export the SDK logger from commerce-sdk.  But we shouldn't confuse the users by adding. a new package that they need to import.

Reply:
the logger key, I have is "COMMERCE_SDK_LOGGER". Do we want to change to "COMMERCER-SDK"?  I prefer having LOGGER in the key though

Reply:
The key is only ever used within the context of the logger. Including LOGGER in the key just seems redundant to me.

Reply:
loglevel.getLogger function returns a new logger if the name does not match. If there is a typo in the logger key, a new logger is returned, no error is thrown.

I think I will export the key and use it